src/Transformer/UuidTransformer.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\Component\Input\Transformer;
+
+use Ramsey\Uuid\Uuid;
+
+class UuidTransformer implements TransformerInterface
+{
+    public function transform($value)
+    {
+        return Uuid::fromString($value);
```

Comment:
See https://github.com/LinioIT/input/blob/master/src/Transformer/DateTimeTransformer.php for the different validations that we have to perform.

Reply:
do you mean we need to fail silently?

for example, if a client app sends an invalid UUID like `'d1d6228d-604c'` -- shouldn't it throw an exception?

Reply:
I think we should probably throw a transformation exception though, and not the exception from the UUID library.